src/types/modules/time_module.cpp

```diff
@@ -22,9 +22,10 @@ TimeModule::TimeModule() : Module("time")
  *
  * @return @c true if success, otherwise @c false.
  */
-bool TimeModule::initialize()
+bool TimeModule::initialize(bool avastSpecific)
 {
 	using Type = Expression::Type;
+	(void) avastSpecific;
```

Comment:
What you can also do in order to avoid unused variable warnings is that you can specify parameter as `bool/* avastSpecific*/`). Basically, don't give a name to the parameter.